packages/nextjs/app/builders/0xF058bE573Fa9303095B9bdED77EE2867d75BbFa6/page.tsx

```diff
@@ -0,0 +1,164 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.
Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.